NAMESPACE

```diff
@@ -8,6 +8,7 @@ export(anc_testing_prev_mf)
 export(artnum_mf)
 export(calculate_prevalence_art_coverage)
 export(calendar_quarter_labels)
+export(calendar_quarter_labels_short)
```

Comment:
Why is this exported here? unrelated change or needed in a dependent hintr pr?

Reply:
Unrelated - the `@export` tag was added to docs at some point but they must not have regenerated

Reply:
ah, ok